packages/frontend/src/stdlib/analyses/sql.tsx

```diff
@@ -0,0 +1,114 @@
+
+/** Button to download an SVG. */
+export default function DownloadTextButton(props: ModelAnalysisProps<DownloadConfig>) {
+    const thSchema = new ThSchema();
+
+    const backends = ["MySQL", "SQLite", "PostgresSQL"];
+
+    const [backend, setBackend] = createSignal("MySQL");
+    const sqlOutput = () => {
+        const model = props.liveModel.elaboratedModel();
+        return model ? thSchema.renderSql(model, backend()) : null;
+    };
+    const downloadText = (text: string) => {
+        downloadTextContent(text, "schema.sql");
+        // TODO get the name of analysis
+    };
+
+    const copyToClipboard = (text: string) => navigator.clipboard.writeText(text);
+
+    const tooltip = () => (
+        <>
+            <p>
+                {
+                    "The following attribute types are parsed as SQL types. Any others are parsed as text."
+                }
+            </p>
+            <ul>
+                <li>{"Int"}</li>
+                <li>{"TinyInt"}</li>
+                <li>{"Float"}</li>
+                <li>{"Bool"}</li>
+                <li>{"Time"}</li>
+                <li>{"Date"}</li>
+                <li>{"DateTime"}</li>
+            </ul>
+        </>
+    );
+
+    const BackendConfig = (backends: string[]) => (
+        <div>
+            <span>Backend: </span>
+            <select
+                value={backend() ?? undefined}
+                onInput={(evt) =>
+                    props.changeContent((content) => {
+                        setBackend(evt.currentTarget.value);
+                        content.backend = backend();
+                    })
+                }
+            >
+                <For each={Array.from(backends)}>
+                    {(bknd) => <option value={bknd}>{bknd}</option>}
+                </For>
+            </select>
+        </div>
+    );
+
+    const title = () => "SQL Schema";
+    const header = (sql: string) => (
+        <div style="display: flex; align-items: center; justify-content: flex-end; gap: 8px; margin-bottom: 8px;">
+            <IconButton
+                onClick={() => copyToClipboard(sql)}
+                disabled={false}
+                tooltip={"Copy SQL to clipboard"}
+            >
+                <Copy size={16} />
+            </IconButton>
+            <IconButton onClick={() => downloadText(sql)} disabled={false} tooltip={""}>
+                <Download size={16} />
+            </IconButton>
+            <IconButton tooltip={tooltip()}>
+                <CircleHelp size={16} />
+            </IconButton>
+        </div>
+    );
+
+    return (
+        <div>
+            <Show when={sqlOutput()}>
+                {(sql) => (
+                    <div>
+                        <Foldable title={title()} header={header(sql())}>
+                            {BackendConfig(backends)}
+                        </Foldable>
+                        <pre style="white-space: pre-wrap;">{sql()}</pre>
```

Comment:
```suggestion
                        <pre style="white-space: pre-wrap; font-family:var(--mono-font)">{sql()}</pre>
```
Changes the font to the one we selected in global.css. This is currently `Roboto Mono` but will change to `Source Code Pro` with #865.

Reply:
Maybe it would be more correct to set the default font-family for `pre` tags in global.css